lib/init-app.js

```diff
@@ -6,7 +6,7 @@ import { uiRouterFactory } from './router/ui-router.factory';
 
 
 
-export default function({ rootPath, serverIoCFactory }) {
+export default function({ rootPath, serverIoCFactory, manifestPath }) {
```

Comment:
let's rename `manifestPath`  into `assetsManifestPath` to provide more explicit meaning.